test/e2e/simple.go

```diff
@@ -43,15 +44,17 @@ func E2ESimple(logger *log.Logger) error {
 	logger.Println("Waiting for 30 seconds to produce blocks")
 	time.Sleep(30 * time.Second)
 
-	blockchain, err := testnode.ReadBlockchain(context.Background(), testNet.Node(0).AddressRPC())
-	testnet.NoError("failed to read blockchain", err)
+	logger.Println("Reading blockchain headers")
+	blockchain, err := testnode.ReadBlockchainHeaders(context.Background(), testNet.Node(0).AddressRPC())
```

Comment:
[nit] related to other comment about renaming function to ReadBlockMetas

```suggestion
	blockMetas, err := testnode.ReadBlockchainHeaders(context.Background(), testNet.Node(0).AddressRPC())
```